lib/src/widgets/help_collection/overlay_flag_help_widget.dart

```diff
@@ -0,0 +1,131 @@
+
+    const mainViewTitleLetterSpacing = 0.2;
+    const mainViewTitleFontSize = 19.6;
+
+    const detailsButtonFontSize = 18.0;
+    const detailsButtonLetterSpacing = -0.2;
+    const detailsButtonSpacingToChevron = 8.0;
+    const detailsButtonSpacingAfterChevron = 6.0;
+
+    return HelpWidget(
+      constraints: const BoxConstraints(
+        maxWidth: _widgetWidth,
+      ),
+      controller: _controller,
+      optionsView: CardRounded(
+        padding: const EdgeInsets.symmetric(vertical: 13),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            DefaultLinksCardWidget(
+              textStyle: theme.textTheme.headline4?.copyWith(
+                fontSize: linkFontSize,
+                color: HelpColors.blue,
+                fontWeight: FontWeight.w500,
+              ),
+              iconTextStyle: const TextStyle(
+                fontFamily: FontConfig.family,
+                fontSize: 16,
+                fontWeight: FontWeight.w400,
+              ),
+              onClose: _controller.goBack,
+            ),
+            const SizedBox(height: 5),
+          ],
+        ),
+      ),
+      mainView: Stack(
+        clipBehavior: Clip.none,
+        children: [
+          CardRounded(
+            closeButtonAlignment: const Alignment(1.02, -1.02),
+            onClose: _controller.goBack,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                const SizedBox(height: 31),
+                Text(
+                  title,
+                  style: Theme.of(context).textTheme.headline6?.copyWith(
+                        fontWeight: FontWeight.w600,
+                        letterSpacing: mainViewTitleLetterSpacing,
+                        fontSize: mainViewTitleFontSize,
+                      ),
+                ),
+                const SizedBox(height: 14),
+                Row(
+                  children: [
+                    const SizedBox(
+                      width: 3,
+                    ),
+                    Expanded(
+                      child: Container(
+                        color: const Color(0xffe5e5e5),
+                        height: 1,
+                      ),
+                    ),
+                    const SizedBox(
+                      width: 3,
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 18),
+                DetailsButton(
+                  title: _defaultDetailsButtonDesc,
+                  color: HelpColors.blue,
+                  onTap: _controller.goForward,
+                  fontSize: detailsButtonFontSize,
+                  letterSpacing: detailsButtonLetterSpacing,
+                  spacingToChevron: detailsButtonSpacingToChevron,
+                  spacingAfterChevron: detailsButtonSpacingAfterChevron,
+                ),
```

Comment:
This looks like it needs some refactoring :)

Reply:
Yeah, probably will remake it to just `textStyle` and `chevronPadding`